reframe/frontend/statistics.py

```diff
@@ -82,7 +82,11 @@ def json(self, force=False):
                 entry = {
                     'build_stderr': None,
                     'build_stdout': None,
-                    'dependencies': [
+                    'dependencies_actual': [
+                        (d.check.name, d.partition.fullname, d.environ.name)
```

Comment:
If I'm not mistaken you can write this simply as `(*d)`.

Reply:
No, ignore this. The correct is `[*d]`, but it wouldn't work, because it returns the objects not the names.